crypto/serializer/serializer_local.h

```diff
@@ -50,3 +72,59 @@ struct ossl_serializer_ctx_st {
      */
     UI_METHOD *allocated_ui_method;
 };
+
+struct ossl_deserializer_instance_st {
+    OSSL_DESERIALIZER *deser;    /* Never NULL */
+    void *deserctx;              /* Never NULL */
+    const char *input_type;      /* Never NULL */
+    const char *output_type;     /* Never NULL */
```

Comment:
This seems to be not quite correct.. I see no refs to output_type (so it will be NULL).

Reply:
Yeah.  It used to have a meaning in earlier iterations, but no longer...

Reply:
Gone